pom.xml

```diff
@@ -91,11 +91,15 @@
     <dependency>
       <groupId>io.jenkins</groupId>
       <artifactId>configuration-as-code</artifactId>
+      <!-- TODO: remove when included in jenkins-bom via parent -->
+      <version>1995.v540b_50a_eb_0c1</version>
```

Comment:
Normally these overrides would be kept in `dependencyManagement` for clarity, but OK.